packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -417,15 +418,46 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
 
   // gestures.
 
+  // Converts the details.consecutiveTapCount from a TapAndDrag*Details object,
+  // which can grow to be infinitely large, to a value between 1 and the supported
+  // max consecutive tap count. The value that the raw count is converted to is
+  // based on the default observed behavior on the native platforms.
+  //
+  // This method should be used in all instances when details.consecutiveTapCount
+  // would be used.
+  static int _getEffectiveConsecutiveTapCount(int rawCount) {
```

Comment:
I think we have a method with the same name in text_selection.dart.  Is it reasonable to share code here?

Reply:
I think I can re-visit this once I implement triple-click gestures. Right now there are some small nuances between how this effective consecutive tap count works on static vs editable text.

Reply:
Sounds good.